LEAF_Request_Portal/admin/Group.php

```diff
@@ -32,13 +32,23 @@ public function __construct($db, $login)
 
     public function addGroup($groupName, $groupDesc = '', $parentGroupID = null)
     {
-        $vars = array(':groupName' => $groupName,
+        /*$vars = array(':groupName' => $groupName,
```

Comment:
We should remove commented-out code wherever possible when it doesn't pertain to a future change we want to make.  It will make sure our code is as clean as it can be.  If needs to be left in, please mark it with a TODO comment before it.

Reply:
Yeah this code snippet should stay because it will be used in the future.